skflow/estimators/dnn.py

```diff
@@ -17,48 +17,18 @@
 
 from sklearn.base import ClassifierMixin, RegressorMixin
 
-from skflow.estimators.base import TensorFlowEstimator
+from skflow.estimators.base import TensorFlowEstimator, ESTIMATOR_COMMON_DOCSTRING
 from skflow import models
+from skflow.util.doc_utils import Appender
 
 
 class TensorFlowDNNClassifier(TensorFlowEstimator, ClassifierMixin):
-    """TensorFlow DNN Classifier model.
```

Comment:
I agree, we definitely need something like Sphinx and host it somewhere, so people can find and browser nice the docs. Ideally examples as well would be hosted with nice explanations, like sklearn does.